substrate/frame/system/src/lib.rs

```diff
@@ -300,7 +312,7 @@ pub mod pallet {
 			#[inject_runtime_type]
 			type RuntimeTask = ();
 			type BaseCallFilter = frame_support::traits::Everything;
-			type BlockHashCount = frame_support::traits::ConstU64<10>;
+			type BlockHashCount = TestBlockHashCount<frame_support::traits::ConstU32<10>>;
```

Comment:
Most mocks had `250` for this. I assume it's good anyway, right?

Reply:
yes, I think 250 was just random copy&pasted value, looks like there is no test depending on this value, so all test passed and should be ok